RecoMuon/GlobalTrackingTools/interface/MuonTrackingRegionBuilder.h

```diff
@@ -62,6 +62,18 @@ class MuonTrackingRegionBuilder : public TrackingRegionProducer {
     /// Add Fill Descriptions
     static void fillDescriptions(edm::ParameterSetDescription& descriptions);
 
+    // 2016-08-10 MK: I'm pretty sure the fillDescriptions() above is
+    // not used in practice by any EDModule (it's called by
+    // L3MuonTrajectoryBuilder::fillDescriptions(), which itself is
+    // not called by anybody). I'm mainly confused that the
+    // fillDescriptions() above adds two PSets
+    // ("MuonTrackingRegionBuilder" and
+    // "hltMuonTrackingRegionBuilder") to the argument PSet, while to

```

Comment:
this was a part of https://github.com/cms-sw/cmssw/pull/8356/files#r28954056.
I couldn't find the inline code discussion on the web: the point for making two were to provide defaults relevant for HLT and for offline. 
The change came with https://github.com/cms-sw/cmssw/pull/8356#issuecomment-97246364
at that point the defaults didn't change and were set with customize.
By now they ended up in HLTPSetMuonTrackingRegionBuilder8356 and a switch to hltMuonTrackingRegionBuilder never really happened.
So, it's a dormant new feature.

@HuguesBrun @BenjaminRS @fwyzard could you please comment

@makortel it may be better to move comments from the inline code to some github issue with maybe shorter note here